common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -162,6 +165,11 @@ public void postCommandResult(@NonNull Runnable runnable) {
         handler.post(runnable);
     }
 
+    @Override
+    public KeyManagerFactory getSslContextKeyManagerFactory() throws NoSuchAlgorithmException {
+        return KeyManagerFactory.getInstance("X509");
```

Comment:
You may want to add a link to documentation for KeyManagerFactory and a description of what's this actually is and what it can be used to accomplish.  In other words, what function(s) it performs for us.